containers/unit-tests/setup.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/sh -ex
+
+personality="$1"
+
+dependencies="\
+    appstream-util \
```

Comment:
Matter for a follow-up: Grab this from tools/debian/control similar to what we do in the image build scripts now, to avoid the duplication. And then just have the extra packages in that list.